quickjs.h

```diff
@@ -1022,7 +1022,6 @@ JS_EXTERN int JS_SetModuleExportList(JSContext *ctx, JSModuleDef *m,
 
 JS_EXTERN const char* JS_GetVersion(void);
 
-#undef JS_EXTERN
```

Comment:
Maybe this is a problem for those modules that dfeine something with the same name?

Since the runner works in Unix only, we could leave this as is perhaps, and drop the `static` ?

Reply:
Yes, fair point, I was ambivalent on whether to make this change or not.

The reason I did is because otherwise it breaks if a downstream user adds `-fvisibility=hidden` to the build flags, builds the shared library, and run-test262. Probably not very likely.

Reply:
Not very likely indeed!

Reply:
From our own CI :see_no_evil:
```
 /usr/bin/ld: CMakeFiles/run-test262.dir/run-test262.c.o: warning: relocation against `js_atomics_mutex' in read-only section `.text'
/usr/bin/ld: CMakeFiles/run-test262.dir/run-test262.c.o: in function `js_agent_sleep':
run-test262.c:(.text+0x45a): undefined reference to `js_atomics_mutex'
```